container.go

```diff
@@ -436,17 +436,11 @@ func (c *Container) fetchState(cmd string) (State, error) {
 	if cmd == "" {
 		return State{}, fmt.Errorf("Cmd cannot be empty")
 	}
-
-	state, err := c.pod.storage.fetchPodState(c.pod.id)
-	if err != nil {
-		return State{}, err
-	}
-
-	if state.State != StateRunning {
+	if c.pod.state.State != StateRunning {
```

Comment:
@jcvenega Sorry but I still don't understand why we need to fetch a container state from a pod that's not running yet. This should not happen, and if it does, the container state should be set to `ready` at most.

In the commit log, you need to explain why we need this, on not only how you do it.

Reply:
@sameo  you was right, I is not needed, my bad,  commit removed.